Gemfile

```diff
@@ -1,3 +1,5 @@
 source 'https://rubygems.org'
-gem 'github-pages', group: :jekyll_plugins
-
+# Bundler 1.x workaround: Remove this when we are on Bundler 2
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+# When this is merged and released, go back to regular gem: https://github.com/github/pages-gem/pull/762
+gem 'github-pages', github: "github/pages-gem", branch: "hubot/kramdown-v2-3-1", group: :jekyll_plugins
```

Comment:
This has since been merged, so what if we instead point towards their master branch?

Reply:
I think we should just wait for a new release: https://github.com/github/pages-gem/releases

shouldn't be that long before it happens?

IIRC this site runs on GitHub Pages so it's not like we control that anyway